### ISSUE TITLE
MFC-T910DW on FreeBSD 12.2-RELEASE

### ISSUE THREAD

**Issue by geoffmx** [pdewacht/brlaser#110]
I have this printer connected over the network so the command below has no output.
# lpinfo --include-schemes usb -l -v

None of the printers listed in cups with brlaser work with the MFC-T910DW.

**Comment by QORTEC** [comment]
@geoffmx you are correct, that command requires the printer to be connected via USB to get any useful information. 
If you could temporally connect your printer via usb and provide output of this command it would be much appreciated.

**Comment by geoffmx** [comment]
Thanks @QORTEC

Here's the output from "sudo lpinfo --include-schemes usb -l -v"

Device: uri = usb://Brother/MFC-T910DW?serial=[REDACTED]
        class = direct
        info = Brother MFC-T910DW
        make-and-model = Brother MFC-T910DW
        device-id = MFG:Brother;CMD:HBP,PJL,URF;MDL:MFC-T910DW;CLS:PRINTER;CID:Brother Generic Jpeg Type2;URF:SRGB24,W8,CP1,IS19-1,MT1-8-11,OB9,PQ4-5,RS200-300,OFU0,V1.4,DM3;
        location =

**Comment by zeroheure** [comment]
@geoffmx didn't reported that it works
@QORTEC are you sure it works ?

**Comment by QORTEC** [comment]
@geoffmx if you can confirm your printer works with this driver it would be appreciated, thought since it's an inkjet I suspect it does not.

@zeroheure Thanks, don't know how I missed that
I'll double check my pull request to see if I made the same error elsewhere.
I'll try to update it 2-3 when I get time off.

**Comment by geoffmx** [comment]
@QORTEC, the MFC-T910DW, as you noted, is an inkjet printer and does not work with brlaser. I can confirm that this printer works on a FreeBSD 13.0-RELEASE via Generic IPP Everywhere.

**Comment by zeroheure** [comment]
@QORTEC my list on the wiki should be up to date for those working and not.

@geoffmx thank you very much for your confirm.
